src/components/editor/property/InputListLOC.jsx

```diff
@@ -82,6 +91,20 @@ const InputListLOC = (props) => {
     changeSelections(payload)
   }
 
+  const lookupCheckboxes = Object.values(allLookupConfigs).map((lookupConfig) => {
```

Comment:
I wonder if it would be worth making a component of this (despite the rule of threes)?

Reply:
Yeah, we do that with SearchResultRows. Not a huge fan, as it makes the code less clear.